### PR TITLE
documentation fix

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/python_demos/image_retrieval_demo/image_retrieval_demo.py [hunk 1]
```diff
@@ -44,7 +44,14 @@ def build_argparser():
...
     args.add_argument('-g', '--gallery',
-                      help='Required. Path to a file listing gallery images.',
+                      help='Required. Path to a text file listing gallery images. The text file must have following format:'
+                      '\n'
+                      '\n    <path_to_image_1> <ID_1>'
+                      '\n    <path_to_image_2> <ID_2>'
+                      '\n    ...'
+                      '\n    <path_to_image_n> <ID_n>'
+                      '\n'
+                      '\nwhere <ID_*> are unique interger numbers.',
```
Comment (IRDonch) [inline demos/python_demos/image_retrieval_demo/image_retrieval_demo.py]: This is IMO not the correct place for these details. It's too verbose for `--help` output, and using `RawTextHelpFormatter` means that none of the help strings will be word-wrapped, which looks worse than the default formatting.

I think it should be explained in the README instead.

Comment (Wovchena) [reply demos/python_demos/image_retrieval_demo/image_retrieval_demo.py]: I suggest leaving links to https://github.com/19900531/test and https://github.com/opencv/openvino_training_extensions/tree/develop/tensorflow_toolkit/image_retrieval/data/gallery as examples in `README.md` in addition.

**Comment by vladimir-dudnik** [comment]
@Ilya-Krylov had you have a chance to see reviewers comments?

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**